front/components/assistant/conversation/ConversationTitle.tsx

```diff
@@ -27,13 +33,32 @@ export function ConversationTitle({
     }, 1000);
   };
 
+  const { conversation, isConversationError, isConversationLoading } =
+    useConversation({
+      conversationId,
+      workspaceId: owner.sId,
+    });
+
+  if (isConversationLoading || isConversationError || !conversation) {
+    return null;
+  }
+
   return (
     <div className="grid h-full max-w-full grid-cols-[1fr,auto] items-center gap-4">
       <div className="overflow-hidden truncate">
-        <span className="font-bold">{title}</span>
+        <span className="font-bold">{conversation?.title || ""}</span>
       </div>
 
-      <div className="flex space-x-1">
+      <div className="flex gap-1">
```

Comment:
likely overflows on mobile ?

Reply:
Here's a mobile screenshot, ok for you?

<img width="618" alt="Capture d’écran 2023-09-26 à 14 17 10" src="https://github.com/dust-tt/dust/assets/3803406/db671123-339e-42fe-b35d-d6a08d2e45e1">

Reply:
I hid them on mobile, and increased a bit the stacking: 

<img width="476" alt="Capture d’écran 2023-09-26 à 14 24 49" src="https://github.com/dust-tt/dust/assets/3803406/8381bbb2-5258-47bd-a2e5-feb30eec57f2">